### ISSUE TITLE
/bid page should distinguish between in-progress and open jobs

### ISSUE THREAD

**Issue by mtlynch** [ReinProject/python-rein#119]
When I view the /bid page, it shows jobs, but in clicking them, some are open for bids and some seem to have a freelancer already assigned. It's not clear if I'm allowed to bid on the ones with a freelancer already assigned, but at the very least, the Rein UI should distinguish between jobs with a bidder assigned and jobs that are still open.

**Comment by weex** [comment]
True, if there's a Rein Offer posted, we shouldn't display in /bid.
